server/routerlicious/packages/services-client/src/error.ts

```diff
@@ -22,10 +22,16 @@ export interface INetworkErrorDetails {
      */
     message?: string;
     /**
-     * Represents the time in milliseconds that should be waited before retrying.
-     * Refer to {@link NetworkError.retryAfterMs}.
```

Comment:
Won't this cause some measure of back-compat problems, to just change the units of this property like this?  And anyway I don't see any code updated to reflect this in this PR...

Reply:
This actually is broken at the moment for this exact reason. #8537 "just changed the units of this property" from seconds to milliseconds. This change brings it back to a working state that is understood by the driver, and adds a new property that we can use to transition without causing back-compat problems, or at least being intentional about it

Reply:
Can you just delete `retryAfter` from here altogether?  The client is only looking for `retryAfterMs` at what comes over the wire.

Reply:
The client is looking for retryAfter
https://github.com/microsoft/FluidFramework/blob/main/packages/drivers/routerlicious-driver/src/restWrapper.ts#L93